node/Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM node:17.6-slim
-RUN npm install --global npm@8.5.1
-
-RUN mkdir -p /home/node/app && chown -R node:node /home/node/app
-COPY /frontend/ /home/node/app/
-WORKDIR /home/node/app
-RUN chown node:node package*.json && rm -rf node_modules
+FROM node:17.6-slim as build
 
 USER node
-RUN npm install && npm run build
-VOLUME /home/node/app/node_modules
+RUN mkdir /home/node/app
+
+WORKDIR /home/node
+COPY /frontend/package.json /frontend/package-lock.json ./
+RUN npm ci
+
+ENV PATH=/home/node/node_modules/.bin:$PATH
+
+WORKDIR /home/node/app
 
-CMD ["npm", "run", "start"]
+CMD ["npm", "run", "start"]
```

Comment:
No newline at EOF :)

Reply:
oh I made copy paste error. But the linter should catch this right?